builder/package.json

```diff
@@ -32,6 +34,10 @@
     "glob": "^11.0.0",
     "next": "14.2.5",
     "tsup": "^8.2.4",
-    "typescript": "^5.5.4"
+    "typescript": "^5.5.4",
+    "vitest": "^2.1.1"
+  },
+  "dependencies": {
+    "ts-morph": "^23.0.0"
```

Comment:
dep or devDep?

Reply:
It does need to be a dependency 🙂 

If we try bundling it in we get this error: https://github.com/flarelabs-net/poc-next/actions/runs/10959287219/job/30431309402